### ISSUE TITLE
This is only the watchOS version of SF

### ISSUE THREAD

**Issue by quinncomendant** [AppleDesignResources/SanFranciscoFont#1]
The README.md states that this font is “used in the Apple Watch, iOS 9, and OS X El Capitan.” That's not true: this is only the Watch version. If you want to download the others, [here's a zip download](https://cl.ly/3h170l1d0X2k/download/SF-Fonts-June19.zip) that contains both. You might want to update this repo with the correct designation, and add the iOS and macOS font as a new repo.

**Comment by manuzark** [comment]
thank you sir

**Comment by ivan** [comment]
You can get newer versions of these fonts directly from Apple at https://developer.apple.com/fonts/:

https://developer.apple.com/fonts/downloads/SFUI.zip
(as of today, .otf timestamps indicate 2016-12-22)

https://developer.apple.com/fonts/downloads/SFCompact.zip
(as of today, .otf timestamps indicate 2017-01-04)

They are in a .pkg file that `xar` doesn't like, though, so good luck extracting them anywhere but a macOS machine.

**Comment by ramadhyay** [comment]
You you can use 7-zip on windows.

Just use "Open Inside" into the payload file and you will find all the UI fonts in .otf format.

**Comment by Grawl** [comment]
I've opened a PR #3 so you can now try new ones

**Comment by piotr-cz** [comment]
@ivan the link to https://developer.apple.com/fonts/downloads/SFUI.zip is broken now

**Comment by Grawl** [comment]
@piotr-cz you always can get new one on https://developer.apple.com/fonts/

**Comment by piotr-cz** [comment]
@Grawl yes, I just found out that there is an official distribution. Unfortunately it comes up in google search unlike this repo

**Comment by ilyoungchung** [comment]
thanks bro
